go/vt/vttablet/tabletmanager/rpc_external_reparent.go

```diff
@@ -135,24 +132,31 @@ func (agent *ActionAgent) TabletExternallyReparented(ctx context.Context, extern
 }
 
 // finalizeTabletExternallyReparented performs slow, synchronized reconciliation
-// tasks that ensure topology is self-consistent, and then marks the reparent as
-// finished by updating the global shard record.
+// tasks that ensure topology is self-consistent
```

Comment:
Can you clean up this paragraph with punctuation and capitalization? It's hard to know when one sentence ends and another begins.